src/main/kotlin/uk/gov/justice/digital/hmpps/healthandmedication/client/prisonersearch/PrisonerSearchClient.kt

```diff
@@ -7,6 +7,10 @@ import org.springframework.web.reactive.function.client.WebClientResponseExcepti
 import uk.gov.justice.digital.hmpps.healthandmedication.client.prisonersearch.dto.PrisonerDto
 import uk.gov.justice.digital.hmpps.healthandmedication.config.DownstreamServiceException
 
+data class PrisonerSearchResult(
```

Comment:
Maybe better to put this in its own file next to the PrisonerDto

Reply:
ah yeah - I meant to do that but fell down a rabbit hole at the time 😅